packages/design-system/src/components/toolbar.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { PlayIcon } from "@webstudio-is/icons";
+import { theme } from "../stitches.config";
+import {
+  Toolbar,
+  ToolbarToggleItem,
+  ToolbarToggleGroup,
+  ToolbarSeparator,
+} from "./toolbar";
+
+export default {
+  title: "Library/Toolbar",
```

Comment:
Just realised: there's no "Toolbar" component in Figma. The idea of `Library/` section is to have a list that matches list in Figma exactly.

I think we should do one of these:
- add Toolbar component in Figma
- have 2 `.stories` files in code: one for `Components/Toolbar` and another for `Library/Toolbar Toggle`

Reply:
yes, @taylornowotny do you mind adding a Toolbar component that you already have as a Component to the library?

Reply:
If there is a radix component, then there should be also a lib component in figma I think

Reply:
Assume we will add the Toolbar to figma

Reply:
> Let's call it "Library/Toolbar, Toolbar Toggle" so that when someone is looking for the "Toolbar Toggle" component from Figma, they will see it in the list.

There is also separator and later other components, so with this logic we would put all of them in there, I think this is not a great idea.

Reply:
We can make clear that these are part of toolbar from figma

Reply:
> There is also separator and later other components, so with this logic we would put all of them in there, I think this is not a great idea.

Yeah, not sure, your call